.github/workflows/deploy-docs-site.yml

```diff
@@ -7,13 +7,20 @@ on:
     paths:
       - "docs/**"
 
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    paths:
+      - "docs/**"
```

Comment:
@sherifnada can you give us a path for the connector docs that we import into docs so we can add it here and on line no. 8?

Reply:
Taras for context, I believe we import connector docs programmatically, but they live outside of the docs folder